src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -988,10 +989,14 @@ hide it and clip it to a square of (width x width) each time width is updated.
             taskProgressView.setGraphicFactory(BackgroundTask::getIcon);
 
             PopOver progressViewPopOver = new PopOver(taskProgressView);
+            if (this.existedProgressViewPopOver == null) {
+                this.existedProgressViewPopOver = progressViewPopOver;
+            }
             progressViewPopOver.setTitle(Localization.lang("Background Tasks"));
             progressViewPopOver.setArrowLocation(PopOver.ArrowLocation.RIGHT_TOP);
-
+            this.existedProgressViewPopOver.hide();
```

Comment:
Another option would be to reuse the existing pop over and only change the content. That is:
```
if (existedProgressViewPopOver == null) {
   existedProgressViewPopOver = new ...
} else {
   existedProgressViewPopOver.setContentNode(taskProgressView).
}
```
This should be a slightly better user experience as there is no flicker of hiding/showing a new pop over.


Then you can also rename the global variable to `progressViewPopOver`

Reply:
In this context, it would be nice if the progress button serves as a toggle: if the pop over is currently shown, and the button is clicked again then the pop over should be hidden.

Reply:
Yes, that's great! I have made the change, it works.